### PR TITLE
Create Database 🅱️

### PR DESCRIPTION
**Pull request by evansnyamekye**
**Database [Vet-Clinic]**
This PR includes the following changes: 

Created a vet-clinic database 🔢. 
Created animal table 🏷️.
insert data into animals tables 🐶. 
Use queries to show animal data ☘️.

1.
vet1
2.
vet2
3.
Vet3
4.
vet4
5.
vet5
6.
vet6
7.
vet7
8.
vet8

Thank you 🛩️.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES 
+('Agumon','2020-02-03', 0, true, 10.23);
+('Gabumon','2018-11-15', 2, true, 8);
+('Pikachu','2021-01-07', 1, false, 15.04);
+('Devimon','2017-05-12', 5, true, 11);
```
Comment (DammyShittu) [inline data.sql]: - [ ] When I try to run your INSERT operation as it is I get errors. Only the first insertion works. You may need to run all the insertions on different lines, like this:

  ```
  INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon','2020-02-03', 0, true, 10.23);

  INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon','2018-11-15', 2, true, 8);
  ```

  image

File: schema.sql [hunk 2]
```diff
@@ -5,8 +5,8 @@ CREATE TABLE animals (
     create_time DATE,
```
Comment (DammyShittu) [inline schema.sql:5]: - **[OPTIONAL]** This `create time` column is not populated in any way in your app so I don't there is any use in creating at all

File: README.md [hunk 3]
```diff
@@ -286,6 +121,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (DammyShittu) [inline README.md:122]: - [ ] Kindly fix your broken MIT link.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @evansnyamekye,

## Highlights
✔️ Table creation

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (evansnyamekye) [reply data.sql:8]: @DammyShittu 
am sorry ..... that will be repetition  of scripts for "INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES" on all of the data I provided. 

I personally don't think you did a good work on my PR.

File: README.md (hunk 3 above)
Comment (evansnyamekye) [reply README.md:122]: Thank you for the MIT feedback ... I will fixed it as requested.

File: schema.sql (hunk 2 above)
Comment (evansnyamekye) [reply schema.sql:5]: @DammyShittu I don't think this SHOULD prevent you from approving my PR.

**Comment by evansnyamekye** [comment]
@DammyShittu All required changes applied and fixed as requested. 

Thank you 🌵.

**Review by skyv26 — APPROVED** [review]
## Hi @evansnyamekye , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implemented the features requested by previous code Reviewer ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**